maintenance/generate-wikimedia-domain-config.js

```diff
@@ -9,7 +9,7 @@ var P = require('bluebird');
 
 var fs = require("fs"),
 	writeFile = P.promisify(fs.writeFile),
-	request = P.promisify(require('request')),
+	request = P.promisify(require('preq/node_modules/request')),
```

Comment:
I was wondering myself why wasn't `preq` used in the first place. Will change that